src/main/java/gov/usgs/cida/pubs/springinit/SecurityConfig.java

```diff
@@ -47,23 +47,23 @@ public void configure(ResourceServerSecurityConfigurer resources) {
 	public void configure(HttpSecurity http) throws Exception {
 		http
 			.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-			.and()
-				.authorizeRequests()
-					//anonymous (public)
-					.antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
-					.antMatchers("/lookup/**", "/publication/**", "/version", "/about/**").permitAll()
-					.antMatchers("/swagger-ui.html", "/webjars/**", "/swagger-resources/**", "/v2/api-docs", "/v3/api-docs").permitAll()
-					//authenticated
-					.antMatchers("/mppublications/*/preview*", "/auth/logout").fullyAuthenticated()
-					//authorized
-					.antMatchers("/**").hasAnyAuthority(configurationService.getAuthorizedAuthorities())
+			.and().authorizeRequests().antMatchers("/**").permitAll();
```

Comment:
We should probably create two different security configs and activate them via spring profiles so we don't accidentally turn of security in production.

Reply:
agreed, the other option would be to figure out Authorization in a dev environment.